src/OpenTelemetry.Extensions.Hosting/OpenTelemetryServicesExtensions.cs

```diff
@@ -35,110 +37,37 @@ public static class OpenTelemetryServicesExtensions
         /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
         public static IServiceCollection AddOpenTelemetryTracing(this IServiceCollection services)
```

Comment:
Curious about the naming convention - do we want `AddOpenTelemetryTracing` or `AddOpenTelemetryTracerProvider`?

Reply:
In a contested mid-year election, `AddOpenTelemetryTracing` won with huge margin against `AddOpenTelemetryTracerProvider` as the later was too lengthy
https://github.com/open-telemetry/opentelemetry-dotnet/issues/1196